### ISSUE TITLE
Question: how do I determine if subscriptions are not active any more?

### ISSUE THREAD

**Issue by crnaosa** [anjlab/android-inapp-billing-v3#299]
Hello everyone,

it was a pleasure to use your library so far :)

Now, after checking purchaseInfo.purchaseData.autoRenewing flag and determining that it is false, I must not remove any content while any user still has an **active subscription** to it, even if that subscription will terminate at the end of the current billing cycle, as is the case since the flag is false.

Is there a way to know if the subscriptions are not active?

Will calling bp.getSubscriptionListingDetails("YOUR SUBSCRIPTION ID FROM GOOGLE PLAY CONSOLE HERE"); return null if the subscription is not active any more?

It's difficult to test it, since I have to wait a day or more until the subscription is not active (according to info from google play store).

Thanks a lot,
Milos

**Comment by Damnum** [comment]
@crnaosa did you ever find out the solution? I don't know how to do this either.

**Comment by crnaosa** [comment]
Not really. If I remember correctly, I came to conclusion that it needs to be handled from the server side.
